packages/runtime/src/components/chakra-ui/NumberInput.tsx

```diff
@@ -55,10 +55,11 @@ const NumberInput: ComponentImplementation<{
   size,
   customerIncrement,
   customerDecrement,
-  mergeState
+  mergeState,
```

Comment:
most change origin from formating, should we add a proper eslintrc/prettierc file here?

Reply:
https://github.com/webzard-io/meta-ui/issues/34